SPECS/srp/srp.spec

```diff
@@ -253,6 +253,9 @@ fi
 %endif
 
 %changelog
+* Wed Feb 19 2025 Chris Co <chrco@microsoft.com>
```

Comment:
Missing version & release. Need to find out why rpmdev-bumpspec didn't automatically add it...